tests/functional/adapter/test_constraints.py

```diff
@@ -1,15 +1,18 @@
 import pytest
 
-from dbt.tests.util import relation_from_name
 from dbt.tests.adapter.constraints.test_constraints import (
     BaseTableConstraintsColumnsEqual,
     BaseViewConstraintsColumnsEqual,
-    BaseConstraintsRuntimeEnforcement
+    BaseIncrementalConstraintsColumnsEqual,
+    BaseConstraintsRuntimeDdlEnforcement,
+    BaseConstraintsRollback,
+    BaseIncrementalConstraintsRuntimeDdlEnforcement,
+    BaseIncrementalConstraintsRollback,
 )
 
 
 _expected_sql_snowflake = """
-create or replace transient table {0} (
+create or replace transient table <model_identifier> (
```

Comment:
The previous version looks like something that gets formatted with `.format()`. Does `<model_identifier>` act like a keyword? Or is it inconsequential to the check? I may not be familiar with this particular functionality.

Reply:
We were having to do some logic to get the model identifier that varied by adapter and materialization.  It's not what we were actually testing so I just replace it.  It's done in the core PR [here](https://github.com/dbt-labs/dbt-core/blob/cd641bfdf0bdb08bc725a71c164b0b6c7fe1f1c6/tests/adapter/dbt/tests/adapter/constraints/test_constraints.py#L218-L223)